metadata-load/src/test/java/uk/ac/ebi/ampt2d/metadata/importer/converter/AnalysisConverterTest.java

```diff
@@ -53,7 +53,7 @@
 
     private static final String ANALYSIS_DOCUMENT_API_XML = "AnalysisDocumentApi.xml";
 
-    private static final String ANALYSIS_DOCUMENT_DATABASE_XML = "AnalysisDocumentDatabase1.xml";
+    private static final String ANALYSIS_DOCUMENT_DATABASE_XML = "EgaAnalysisDocumentDatabase.xml";
```

Comment:
Why replacing the file? Couldn't it be useful to have both, and using the EGA one just for specific checks?

Reply:
I thought we are hitting the database specifically for Ega so renamed the XML, ya we could have both as well.

Reply:
You're right about that, which means the analysis previously queried didn't belong to EGA, right?

Reply:
okies will do that then.